engine/gamesys/src/gamesys/resources/res_texture.cpp

```diff
@@ -494,6 +496,9 @@ namespace dmGameSystem
             if (r == dmResource::RESULT_OK)
             {
                 dmResource::SetResource(params->m_Resource, texture_res);
+
+                texture_res->m_OriginalWidth = dmGraphics::GetOriginalTextureWidth(graphics_context, texture_res->m_Texture);
+                texture_res->m_OriginalHeight = dmGraphics::GetOriginalTextureHeight(graphics_context, texture_res->m_Texture);
```

Comment:
I opted for using `dmGraphics::GetOriginalTexture*`, as it may be that the backend may choose what to return.